include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -619,8 +619,7 @@ class CommonOperatorLoader : public ProtobufLoader {
     return llvm::Error::success();
   }
 
-  llvm::Error loadReduceMeanOrSum(llvm::StringRef typeName, const OpType &op,
-                                  ArgumentDictionaryTy &dict) {
+  llvm::Error loadReduceSum(const OpType &op, ArgumentDictionaryTy &dict) {
```

Comment:
If we move `keepDims` into `createBatchedReduceAdd()` like you did for `createBatchedReduceMean()`, can't we keep these two cases merged together? I think the logic would again be the same and so it's better than duplicating the case.

Reply:
That would require some more implementation changes, and testing that we didn't consider as they were not on our path. Also, we were trying focusing on additions, not modifications of the original code, thus the split, to keep ReduceSum as is. If you want that change as a part of this PR I can do that. Let me know.

Reply:
Yes, I would say change it here in this PR. Our unit tests should make sure other things didn't break (I hope... 😃).

Reply:
Done